dali/pipeline/operators/crop/basic/crop.h

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
We need to have GPU crop in mind as well. Putting CPU implementation in the .h file is the bad idea.

Reply:
This just extracted the 'Kernel' for the CPU. I think, we can handle GPU by flattening the Batch tensor from [Samples, Frames, H, W, C] -> [Samples * Frames, H, W, C] and reverting it after the crop is done.